### ISSUE TITLE
pt2onnx---export.py

### ISSUE THREAD

**Issue by gottastudy** [bubbliiiing/yolov5-pytorch#126]
老师，您好!
请问可否出一个export.py文件，实现将
 torch.save(model, os.path.join(save_dir, "ep%03d-loss%.3f-val_loss%.3f.pt" % (epoch + 1, loss / epoch_step, val_loss / epoch_step_val)))
保存的.pt文件转换为.onnx文件。
万分感谢！

**Comment by gottastudy** [comment]
注意到您将源码的yolov5s.yaml文件修改为了YoloBody类。在使用官方提供的export.py文件导出时出现AttributeError: 'YoloBody' object has no attribute 'get'错误。请问有比较好的解决思路吗？
